mobile/src/stateMachine/views/qol/startQOL.tsx

```diff
@@ -6,7 +6,7 @@ import { StyleSheet, Text } from 'react-native';
 import { MasloPage, Container, Button } from 'src/components';
 import { ScenarioTriggers } from '../../abstractions';
 import AppController from 'src/controllers';
-import { QolType } from 'src/viewModels/QoLViewModel';
+import { QolType } from 'common/models/QoL';
```

Comment:
since this isn't a database model, can we move it to the mobile constants directory